### ISSUE TITLE
Fastlane is erroneously warning me I don't have any code signing identities

### ISSUE THREAD

**Issue by puls** [fastlane/fastlane#3769]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

Fastlane is warning me that I have no local code signing identities because [it's searching for the string '0 valid identities found' in the output from `security find-identity`](https://github.com/fastlane/fastlane/blob/a6641e6e4b990cbf7e421384ada0f11b21d42930/fastlane_core/lib/fastlane_core/cert_checker.rb#L22). I have 10 valid identities, so the check is passing because the output '10 valid identities found' contains the target string.

```
[15:46:41]: Fetching profiles...
[15:46:44]: Looks like there are no local code signing identities found, you can run `security find-identity -v -p codesigning` to get this output. Check out this reply for more: https://stackoverflow.com/questions/35390072/this-certificate-has-an-invalid-issuer-apple-push-services
```

### Configuration Files

Please copy the complete content of your `Fastfile` and any other configuration files you use below:

Fastfile:
```ruby
(empty)
```

### Environment

fastlane version (fastlane -v): 1.66.0
Do you use bundler, rbenv or rvm? no

**Comment by calebhicks** [comment]
I've seen the same issue over the last 2 days.

**Comment by KrauseFx** [comment]
Do you still have the expired WWDR certificate installed? Did you follow https://stackoverflow.com/questions/35390072/this-certificate-has-an-invalid-issuer-apple-push-services?

**Comment by puls** [comment]
Hi @KrauseFx! Sorry if my explanation wasn't clear enough, but this has nothing to do with the code signing infrastructure itself, rather a small logic error on parsing output from a command. See #3823.

**Comment by mfurtak** [comment]
Thanks for reporting this! The fix for this went in as part of #3853, which will go out as part of the next release! :tada:
